docs/reference/commandline/run.md

```diff
@@ -685,6 +685,7 @@ $ docker run -d -u daemon --ulimit nproc=3 busybox top
 The 4th container fails and reports "[8] System error: resource temporarily unavailable" error.
 This fails because the caller set `nproc=3` resulting in the first three containers using up
 the three processes quota set for the `daemon` user.
+WARNING: Feature nproc limit is only available in Linux kernel 4.3+
```

Comment:
Here we mentioned 4.3+, while in code we compare with 3.10. Could you share more, and I am not sure whether I miss something.